Multiple Service CIDRs - KEP 1880

aojea opened the pull request:
<!--
 Hello!

 PLEASE title the FIRST commit appropriately, so that if you squash all
 your commits into one, the combined commit message makes sense.
 For overall help on editing and submitting pull requests, visit:
  https://kubernetes.io/docs/contribute/suggesting-improvements/

 Use the default base branch, “main”, if you're documenting existing
 features in the English localization.

 If you're working on a different localization (not English), see
 https://kubernetes.io/docs/contribute/new-content/overview/#choose-which-git-branch-to-use
 for advice.

 If you're documenting a feature that will be part of a future release, see
 https://kubernetes.io/docs/contribute/new-content/new-features/ for advice.
-->
### Description

Multiple Service CIDRs #1880 - GA

<!--
 Remember to ADD A DESCRIPTION and delete this note before submitting
 your pull request. The description should explain what will change,
 and why.
-->

### Issue

https://github.com/kubernetes/enhancements/issues/1880

<!--
 If this pull request resolves an open issue, please link the issue in the PR
 description so it will automatically close when the PR is merged.

 See the GitHub documentation for more details and other options:

 https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
-->

Closes: #

aojea commented:
/assign @thockin @danwinship

thockin reviewed (commented):
Thanks!

/lgtm
/approve

tengqm commented on content/en/docs/reference/command-line-tools-reference/feature-gates/DisableAllocatorDualWrite.md:
```diff
@@ -9,6 +9,9 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.31"
```
```suggestion
    fromVersion: "1.31"
    toVersion: "1.32"
```

tengqm commented on content/en/docs/reference/command-line-tools-reference/feature-gates/MultiCIDRServiceAllocator.md:
```diff
@@ -13,5 +13,8 @@ stages:
   - stage: beta
     defaultValue: false
     fromVersion: "1.31"
```
```suggestion
    fromVersion: "1.31"
    toVersion: "1.32"
```

tengqm commented on content/en/docs/reference/networking/virtual-ips.md:
```diff
@@ -517,6 +514,46 @@ kubernetes       10.96.0.0/28  17m
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "servicecidrs.default"
```
please don't use "." in object name.
it doesn't look like a good practice.

aojea replied:
```suggestion
  name: "servicecidrs-default"
```

aojea commented on content/en/docs/reference/networking/virtual-ips.md:
```diff
@@ -517,6 +514,46 @@ kubernetes       10.96.0.0/28  17m
+metadata:
+  name: "servicecidrs-binding"
+spec:
+  policyName: "servicecidrs.default"
```
```suggestion
  policyName: "servicecidrs-default"
```

aojea commented:
> Hello @aojea 👋! I'm reaching out from the Docs team. Just checking in as we approach **Docs Freeze** on **8th April, 2025 18:00 PDT**. This documentation appears to still be under review. To meet the Docs Freeze, this PR must have a technical review as well as `lgtm` and `approve` labels applied, without any unaddressed comments or concerns from SIG Docs. The status of this enhancement is marked as `At risk for docs freeze`. Thank you!

@tengqm the technical review is already approved, and you are comments are already addresssed

aojea commented:
the lgtm got dropped on the updates to the comments, can someone reapply it? there was no change on the technical content

tengqm commented:
/lgtm
